README.md

```diff
@@ -164,3 +168,10 @@ l * incorrect_above + (1-l) * correct_below
 certain threshold and `correct_below` describes the fraction of correct 
 predictions below a certain threshold. Threshold and `l` can be configured 
 through environment variables.
+
+# Github Action
```

Comment:
One more thought: GitHub steps are killed after 300 minutes. A hyperparameter search is likely to take longer. We should either document that or explicitly show the usage of `timeout-minutes` in our workflow which increases the default timeout.

Reply:
@wochinge it seems each job has a hard limit of 360 minutes; this can't be increased even on paid plans afaik.

Reply:
wow. thanks for researching! The use of the github action is somewhat limited then 😢

Reply:
yeah... there were several complaints of that kind out there =/